### DISCUSSION TITLE
Popover

### DISCUSSION THREAD

**Discussion by d-oit** [Megabit/Blazorise#2354] [General]
There may already be a component in Blazorise ->  https://mudblazor.com/components/popover 
Maybe under a different name? The bootstrap popover isn't pretty with Blazor - always have to init each via JS :-(

**Comment by stsrki** [comment]
We don't have a popover as you posted it. The closest is the Tooltip component https://bootstrapdemo.blazorise.com/tests/elements

**Reply by d-oit** [reply]
Thanks for your quick answer. It's simple with popperjs - but the mudblazor implentation with css looks much better. 

Started with syncfusion blazor 1 year ago and the performance was really bad. 
Have to find some time for it. 
Maybe you could started a twitch show or meetup for new components. 
Then I have no more excuses :-)

**Reply by stsrki** [reply]
In the latest 0.9.4-preview, I have implemented tippy.js as the main Tooltip provider. It seems it can be used to add custom HTML as a content: https://atomiks.github.io/tippyjs/v6/html-content/

**Reply by atibilov** [reply]
Hello
I'm trying to create popover (like GitHub popover when mouse under the username)
but unfortunately, don't understand what I need to do
an error 
 	Unrecognized child content inside component 'Tooltip'. The component 'Tooltip' accepts child content through the following top-level items: 'ChildContent'.		 

     <Tooltip Interactive=true>
         @UserName
         <ChildContent>
          @render //renderfragment
         </ChildContent>
     </Tooltip>

in official [docs ](https://blazorise.com/docs/components/tooltip) i don't find answer/example
please help

**Reply by stsrki** [reply]
We need to add it to the documentation, but there is an example in the demo app. See it at https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/TooltipsPage.razor#L72

**Reply by atibilov** [reply]
thank you so much for the quick response!
